test/blackbox-tests/test-cases/coq/coqtop-root.t/run.t

```diff
@@ -14,5 +14,5 @@ This test is currently broken due to the workspace resolution being faulty #5899
   1 | (coq.theory
   2 |  (name foo))
   Error: 'coq.theory' is available only when coq is enabled in the dune-project
```

Comment:
```suggestion
  Error: 'coq.theory' is available only when Coq is enabled in the ``dune-project``
```

Reply:
I would rather keep the lower case for this message since it is really saying when `coq lang` is enabled.